src/Widgets/DisplayWidget.vala

```diff
@@ -49,7 +55,7 @@ public class DisplayWidget : Gtk.Grid {
             /* Ignore horizontal scrolling on wingpanel indicator */
             if (e.direction != Gdk.ScrollDirection.LEFT && e.direction != Gdk.ScrollDirection.RIGHT) {
                 /* Determine whether scrolling on mic icon or not */
-                if (show_mic && e.x < mic_icon.pixel_size + mic_icon.margin_end) {
+                if (show_mic && e.x < 24 + mic_icon.margin_end) {
```

Comment:
It's not possible to retrieve the with from the Gtk.Spinner? 
Or would using the `Gtk.IconSize.LARGE_TOOLBAR` (in combination with something like icon_size_lookup) be an option?